lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2060,16 +2060,19 @@ SwiftASTContext::CreateInstance(lldb::LanguageType language, Module &module,
     // swift::ASTContext
     Progress progress("Importing Swift standard library");
     swift_ast_sp->m_ast_context_ap->SetPreModuleImportCallback(
-        [&progress](llvm::StringRef module_name, bool is_overlay) {
-          progress.Increment(1, (is_overlay ? module_name.str() + " (overlay)"
-                                            : module_name.str()));
+        [&progress](llvm::StringRef module_name,
+                    swift::ASTContext::ModuleImportKind kind) {
+          progress.Increment(1, (kind == swift::ASTContext::Overlay
```

Comment:
Should we have a switch here now that we have 3 kinds of module imports ?

Reply:
Honestly I think even the overlay case makes no sense here since the stdlib is always a pure Swift module. I just left it in because it was there before.

Reply:
Fair